client/components/explorer/sidebar/sidebar-tabs-directive.js

```diff
@@ -38,7 +39,7 @@ explorer.components.explorer.sidebar.SidebarTabsDirective = function() {
     controller: [
         '$scope', 'dashboardService', 'sidebarTabService',
         function($scope, dashboardService, sidebarTabService) {
-      /** @export {!DashboardService} */
+      /** @export */  // Cannot use real type DashboardService due to circular dependency.
```

Comment:
Fixed - I can't reproduce the circular dependency, maybe I had mistyped the goog.require() when I had initially tried to use a known type here.